UPGRADING.md

```diff
@@ -2,6 +2,10 @@
 
 This document describes breaking changes and how to upgrade. For a complete list of changes including minor and patch releases, please refer to the [changelog](CHANGELOG.md).
 
+## v3
+
+Dropped node 9.
```

Comment:
Same here. I thought we decided to bump major when dropping even node versions. I.e. we still support node 8 here.